docs/content/documentation/content/image-processing/index.md

```diff
@@ -50,7 +50,7 @@ The function returns a full URL to the resized image.
 
 ## Resize operations
 
-The source for all examples is this 300 × 380 pixels image:
+The source for all examples is this 300 pixel × 380 pixel image:
```

Comment:
is 300x380 pixels not more correct than 300 pixel x 380 pixel? No s on pixel?

Reply:
Here, "pixel" is an adjective. Similar to "3-meter-tall building" (not "3-meters-tall building"). I omitted "-" here because "300-pixel x 380-pixel image" is difficult to read.